contracts/staking/SimpleStaking.sol

```diff
@@ -424,17 +417,21 @@ contract SimpleStaking is ERC20Upgradeable, DAOContract, BaseShareField {
 
 	/**
 	 * @dev Calculates the current interest that was gained.
-	 * @return (uint256, uint256, uint256) The interest in iToken, the interest in USD
+	 * @return (uint256, uint256, uint256) The interest in iToken, the interest in Token and the interest in USD
 	 */
 	function currentUBIInterest()
```

Comment:
this seems  like it can be merged to currentGains

Reply:
Removed and integrated it in currentGains